src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/help/Get-AzureRmDataLakeAnalyticsJobPipeline.md

```diff
@@ -110,7 +115,6 @@ System.Guid
 
 ## OUTPUTS
 
-### System.Collections.Generic.List`1[[Microsoft.Azure.Commands.DataLakeAnalytics.Models.PSJobPipelineInformation, Microsoft.Azure.Commands.DataLakeAnalytics, Version=3.0.0.0, Culture=neutral, PublicKeyToken=null]]
```

Comment:
That's what I did, it's a couple lines lower in the MD and that's why you can see it initially...

![image](https://user-images.githubusercontent.com/14112499/28098203-ca3d37d0-6669-11e7-8b61-72bdd2b084d4.png)